content/docs/openapi/orchestrator-openapi-spec.md

```diff
@@ -0,0 +1,101 @@
+---
+date: "2024-03-08"
+title: Orchestrator plugin OpenAPI
+---
+
+Orchestrator plugin is composed by three parts:
```

Comment:
pls add links to each plugin in janus-idp/plugins

Reply:
Addressed :)